vmm/src/vm.rs

```diff
@@ -2487,6 +2495,11 @@ impl Pausable for Vm {
         self.cpu_manager.lock().unwrap().pause()?;
         self.device_manager.lock().unwrap().pause()?;
 
+        #[cfg(feature = "mshv")]
```

Comment:
Since there is a default implementation - you don't need to make this conditional - applies to the other changes in this file too.